scripts/build_docker_builder.sh

```diff
@@ -50,8 +50,8 @@ if [ $REPO_PROVIDER === "aws" ]
 then
   if [ $PRIVATE_REPO == "true" ]
   then
-    node ./scripts/prune_ecr_images.js --repoName $REPO_NAME-builder --region $REGION --service builder --releaseName $STAGE
+    npx ts-node ./scripts/prune_ecr_images.ts --repoName $REPO_NAME-builder --region $REGION --service builder --releaseName $STAGE
   else
-    node ./scripts/prune_ecr_images.js --repoName $REPO_NAME-builder --region us-east-1 --service builder --releaseName $STAGE --public
+    npx ts-node ./scripts/prune_ecr_images.ts --repoName $REPO_NAME-builder --region us-east-1 --service builder --releaseName $STAGE --public
```

Comment:
Public repos can be in any region. Should we use `$REGION` here as well?

Reply:
I swear that every time I've tried to access a public repo that's located in a different region using that region, it hasn't worked, but that using us-east-1 as the region has worked. I will re-check this to confirm.

Reply:
Yeah, the SDK for ECR Public works a little weird when combined with how public repos are addressed. I think public repos are region-less, or are always in us-east-1; if I switch regions in the web console when looking at public repos, I always see every one.

If I set up ECRPUBLICClient with a region other than us-east-1, then try to call a DescribeImagesCommand, I get e.g. 
```
Uncaught Error: getaddrinfo ENOTFOUND api.ecr-public.us-west-1.amazonaws.com
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:107:26)
    at GetAddrInfoReqWrap.callbackTrampoline (node:internal/async_hooks:128:17) {
  errno: -3008,
  code: 'ENOTFOUND',
  syscall: 'getaddrinfo',
  hostname: 'api.ecr-public.us-west-1.amazonaws.com',
  '$metadata': { attempts: 1, totalRetryDelay: 0 }
}
```
I don't know why it lets you change the region if anything other than us-east-1 is invalid, but that's why I hard-coded it to us-east-1.

Reply:
that's very weird. always fun exploring the hidden functionality of AWS